app/src/main/java/com/example/bigowlapp/activity/HomePageActivity.java

```diff
@@ -79,9 +80,15 @@ protected void initialize() {
         imgBtnOverflow = findViewById(R.id.action_overflow);
         imgBtnOverflow.setOnClickListener(v -> {
             PopupMenu popup = new PopupMenu(this, v);
-            popup.getMenu().add(Menu.NONE, View.generateViewId(), 3, "Edit Profile");
             popup.setOnMenuItemClickListener(this);
             popup.inflate(R.menu.big_owl_overflow);
+            for (int i = 0; i < popup.getMenu().size(); i++) {
```

Comment:
Thanks for addressing my change. Also much better approach since it will always add the edit profile after refresh, no matter how we mess with the popup menu.